docs/content/tutorial/setup.mdx

```diff
@@ -37,20 +24,45 @@ To verify that your Dagster installation was successful, it's time for you to cr
 To create the project, run:
 
 ```bash
-dagster project scaffold --name tutorial-project
+dagster project from-example --example tutorial --name tutorial-project
 ```
 
-To verify that it worked and that you can run Dagster locally, run:
+After running this command, you should see a new directory called `tutorial-project` in your current directory. This directory contains the files that make up your Dagster project. Next, you'll install the Python dependencies you'll be using during the tutorial.
 
 ```bash
 cd tutorial-project
+pip install -e ".[dev]"
+```
+
+This command also installs packages that aren't necessary for every Dagster project but are used for this tutorial. You don't have to read up on them, but if you're curious:
+
+- `requests` will be used to download data from the internet
+- `pandas` is a popular library for working with tabular data
+- `matplotlib` makes it easy to make charts in Python
+- `dagster_duckdb` manages how Dagster can read and write to DuckDB, an in-memory data warehouse similar to SQLite, that you'll use for this tutorial
+- `dagster_duckdb_pandas` allows loading data from DuckDB into Pandas DataFrames
+- `Faker` is a library for generating fake data
+
+<Note>
+  The `-e` flag installs the project in [editable
+  mode](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs),
+  which means that most changes you make in your Dagster project will
+  automatically be applied. The main exceptions are when you're adding new
+  assets or installing additional dependencies.
+</Note>
```

Comment:
```suggestion
<Note>
  The <code>-e</code> flag installs the project in <a href="https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs">editable
  mode</a>,
  which means that most changes you make in your Dagster project will
  automatically be applied. The main exceptions are when you're adding new
  assets or installing additional dependencies.
</Note>
```

⛏️ Note components have to use HTML formatting. For now!

❓ Should this callout be moved up, perhaps so it follows the command itself? It coming after the list of libraries is a little odd to me.